cmd/daytona/config/const.go

```diff
@@ -50,6 +50,7 @@ func GetSupportedGitProviders() []GitProvider {
 		{"bitbucket", "Bitbucket"},
 		{"codeberg", "Codeberg"},
 		{"gitea", "Gitea"},
+		{"gitness", "GitNess"},
```

Comment:
`Gitness` instead of `GitNess`

Reply:
sure i will apply all change in few minutes